x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecurityRestIT.java

```diff
@@ -84,6 +85,17 @@ public class RemoteClusterSecurityRestIT extends AbstractRemoteClusterSecurityTe
             .build();
     }
 
+    /**
+     * This is needed because `randomBoolean` is not available inside the outer `static {}` scope,
+     * where the initial cluster settings are set
+     */
+    private static String atomicallyRandomizeSslEnabled(AtomicReference<Boolean> sslEnabledRef) {
```

Comment:
I'm passing the atomic ref explicitly since it reads a little clearer to me this way where the "atomically" comes from. No strong preference though, can also inline.